applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, Model, settings):
         default_settings = KratosMultiphysics.Parameters(r'''{
             "model_part_name": "",
             "wake_direction": [1.0,0.0,0.0],
+            "create_output_file": false,
```

Comment:
Also this is not needed anymore, right?

Reply:
Yes I think this can be removed! I 'll do it.